src/yang/ietf-bgp-policy.yang

```diff
@@ -248,6 +248,32 @@ module ietf-bgp-policy {
           }
         }
       }
+      container next-hop-sets {
+        description
+          "Definition of a list of IPv4 or IPv6 next-hops which can
+           be matched in a routing policy.";
+
+        list next-hop-set {
+          key "name";
+          description
+            "List of defined next-hop sets for use in policies.";
+
+          leaf name {
+            type string {
+              length "1..79";
+            }
+            mandatory true;
+            description
+              "Name of the next-hop set.";
+          }
+          leaf-list address {
```

Comment:
Should this simply be "next-hop"?
I don't know that we have a pattern for the policy sets yet.  For example, if the match container is «thing»-set, then the individual elements perhaps should be «thing»?

Reply:
What we have for a pattern is

```
container blah-sets {

  list blah-set {

    leaf-list <whatever best describes the list> {

    }
  }
}
```

which we could change to:

```
container blah-sets {

  list blah-set {

    leaf-list blah {

    }
  }
}
```

But we would have to do it for all cases in that case.